src/SFA.DAS.EmployerAccounts.Jobs/StartupJobs/CreateReadStoreDatabaseJob.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections.ObjectModel;
+using System.Threading.Tasks;
+using Microsoft.Azure.Documents;
+using Microsoft.Azure.Documents.Client;
+using Microsoft.Azure.WebJobs;
+using SFA.DAS.EmployerAccounts.ReadStore.Data;
+
+namespace SFA.DAS.EmployerAccounts.Jobs.StartupJobs
+{
+    public class CreateReadStoreDatabaseJob
+    {
+        private readonly IDocumentClient _documentClient;
+
+        public CreateReadStoreDatabaseJob(IDocumentClient documentClient)
+        {
+            _documentClient = documentClient; 
+        }
+
+        [NoAutomaticTrigger]
+        public async Task Run()
+        {
+            var database = new Database
+            {
+                Id = DocumentSettings.DatabaseName
+            };
+
+            var documentCollection = new DocumentCollection
+            {
+                Id = DocumentSettings.AccountUsersCollectionName,
+                PartitionKey = new PartitionKeyDefinition
+                {
+                    Paths = new Collection<string>
+                        {
+                            "/accountId"
+                        }
+                },
+                UniqueKeyPolicy = new UniqueKeyPolicy
+                {
+                    UniqueKeys = new Collection<UniqueKey>
+                        {
+                            new UniqueKey
+                            {
+                                Paths = new Collection<string> { "/userRef" }
+                            }
+                        }
+                }
+            };
+
+            await _documentClient.CreateDatabaseIfNotExistsAsync(database);
+            await _documentClient.CreateDocumentCollectionIfNotExistsAsync(UriFactory.CreateDatabaseUri(database.Id), documentCollection);
```

Comment:
Can `OfferThroughput` be set to `1000` here? It's a default that DevOps would like us to stick to.